src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -20,9 +20,9 @@ internal enum TypeCompareKind
         IgnoreDynamicAndTupleNames = IgnoreDynamic | IgnoreTupleNames,
 
         IgnoreNullableModifiersForReferenceTypes = 8,
-        UnknownNullableModifierMatchesAny = 16,
+        ObliviousNullableModifierMatchesAny = 16,
```

Comment:
I recognize this is an internal API so we can change easily, but we should clarify if we want "unknown" or "oblivious".
If we stick with "oblivious" terminology, then we should close https://github.com/dotnet/roslyn/issues/29632

Reply:
I'd rather not have "Unknown", "Oblivious", and "Disabled" all for the same concept.  This advances (completes?) the removal of one of them.